package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "compile": "hardhat compile",
     "compile:harnesses": "env SRC=./certora/harnesses hardhat compile",
-    "coverage": "env COVERAGE=true hardhat coverage",
+    "coverage": "env COVERAGE=true hardhat coverage && FOUNDRY_FUZZ_RUNS=10 forge coverage --report lcov && lcov -o lcov.info --remove lcov.info 'test/*' 'contracts/mocks/*' && awk '!/,0/' lcov.info > temp && mv temp lcov.info",
```

Comment:
If possible I'd like to avoid having to much logic in the package.json

Is it possible to have a specific profile in the `foundry.toml` file that specify this number of runs ? IMO it would be better if we could remove it from this line.

Also, I'm wondering it we should put that logic in a `./script/coverage.js`.

Reply:
[The documentation](https://book.getfoundry.sh/reference/cli/forge/coverage?highlight=coverage#forge-coverage) mentions a `--ir-minimum` option that I suspect will be needed when doing founcry coverage or P256 (that is already at the limit of stack too deep).

Reply:
yes, originally I added `--ir-minimum` but I did a test with P256 and it wasn't necessary so I removed it -- maybe we have to bring it back, although it increases time by a good amount.